docs/maui/behaviors/icon-tint-effect-behavior.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IconTintColorBehavior - .NET MAUI Community Toolkit
+author: vhugogarcia
+description: "The IconTintColorBehavior allows you to tint an image with an attached property."
+ms.date: 06/27/2022
+---
+
+# IconTintColorBehavior
+
+The `IconTintColorBehavior` is a `behavior` allows you to tint an image with an attached property.
```

Comment:
This might be minor but does it work as an attached property?

e.g.
```xaml
 <Image 
    Source="shield.png"
    toolkit:IconTintColorBehavior.TintColor="Red" />
```

And if it doesn't it would be cool if it did. @pictos is this something we could/should allow?